page_header.html

```diff
@@ -1,4 +1,12 @@
-This information may not apply to the latest version of Elasticsearch.
-For the most up to date information, see the current version of the
-<a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
-Elasticsearch Reference</a>.
+<p>
+  <strong>WARNING</strong>: This documentation covers Elasticsearch 2.x. The 2.x
```

Comment:
> This documentation covers Elasticsearch 2.x.

I'll remove this sentence in the 2.x and 1.x backports, where the version is more obvious.